### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by yersel500**
I develop a website related to "Crypto Congress" with the next features:

- Actually, the webpage has two pages: Home and About pages.
- The speaker section was created dynamically with javascript.
- The mobile version has a hamburger menu for better navigation.

To build the project, I use:

- HTML
- CSS
- Javascript.

Here is a video where I explain the project:

https://www.loom.com/share/38c974bd76f948f693195491251ac9de

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+- javascript
+- linters
+
+## Live Demo
+
+[Click here to see the Portfolio Web](https://yersel500.github.io/capstone-project-yh/)
+
+![image](https://image.prntscr.com/image/ydxJGxH3SNC520UtsK11gw.png)
+
+[Video of the project](https://www.loom.com/share/38c974bd76f948f693195491251ac9de)
```
Comment (rustam-tolipov) [inline README.md:18]: ## GOOD JOB SO FAR 🙌
- [x] [OPTIONAL] Your presentation is awesome and also helped me review your code better 😀. Keep on creating these kinds of great projects 🥇

File: about.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <ul class="menu">
+      <li class="list1"><a class="link1" href="./index.html">Home</a></li>
+      <li class="list1"><a class="link1" href="./about.html">About</a></li>
+    </ul>
+    <div class="hamburger">
+      <span class="line"></span>
+      <span class="line"></span>
+      <span class="line"></span>
+    </div>
+  </nav>
```
Comment (rustam-tolipov) [inline about.html]: - [x] As a requirement you should personalize the content by following the design ⚠. There is no top black part of the navigation on your website. Let's add to fix this issue 😉. And also don't forget to hide it on the mobile version.

#### Original design
image

#### Your website missing part
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  </section>
+
+  <!-- speaker section -->
+  <section class="speakers">
+    <h2 class="featured-speakers">Featured Speaker</h2>
+    <div class="card-speaker-container">
+     
+    </div>
+    <button class="show-more">MORE</button>
+
```
Comment (rustam-tolipov) [inline index.html:95]: - [x] [OPTIONAL] There is one little thing I realized is in this button. Please also change the text `MORE` to `LESS` when it opened 🙌

#### Screenshot 
image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @yersel500  , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Dry code 💯
- Design is beautiful ✔


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hey @yersel500, I am **Hamza** your reviewer for this PR 😸 

## Congratulotions, Project Approved 🟢 

_You did a great job Yersel, I really enjoyed surfing on your website 😃_ 

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ Great Job building your project with **Mobile-First**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**